pkg/boot/syslinux/syslinux.go

```diff
@@ -87,12 +88,18 @@ func ParseLocalConfig(ctx context.Context, diskDir string) ([]boot.OSImage, erro
 //
 // `s` is used to fetch any files that must be parsed or provided.
 //
-// `wd` is the default scheme, host, and path for any files named as a
-// relative path - e.g. kernel, include, and initramfs paths are requested
-// relative to the wd.
-func ParseConfigFile(ctx context.Context, s curl.Schemes, url string, wd *url.URL) ([]boot.OSImage, error) {
-	p := newParser(wd, s)
-	if err := p.appendFile(ctx, url); err != nil {
+// rootdir is the partition mount point that syslinux is operating under.
```

Comment:
This commit about the rootdir, can it be in a separate PR?

Reply:
No, it's a fix necessary for the Debian 9 image to work. In fact, I update the JSON for the image here: https://github.com/u-root/u-root/pull/1696/commits/e62e4ef0bd9869de38e6c796abfe4d3ddb14e9cf